ospf6d/ospf6_network.c

```diff
@@ -73,21 +74,29 @@ static void ospf6_set_checksum(void)
 #endif /* DISABLE_IPV6_CHECKSUM */
 }
 
-void ospf6_serv_close(void)
+void ospf6_serv_close(int ospf6_sock)
```

Comment:
Suggestion: how about making this function prototype into: `void ospf6_serv_close(int *ospf6_sock)`?

Then you can close the fd and set it to `-1` avoiding to have us remember to set it to an invalid value everywhere this function is called.

Reply:
Got it. 
yes, the suggestion is absolutely correct. 
Thanks @rzalamena for the review. 
Will apply the suggested changes.